src/providers/local/join-pool.provider.ts

```diff
@@ -71,6 +72,7 @@ const provider = (props: Props) => {
   const priceImpact = ref<number>(0);
   const highPriceImpactAccepted = ref<boolean>(false);
   const txError = ref<string>('');
+  const approvalActions = ref<TransactionActionInfo[]>([]);
```

Comment:
Should we not use these in the join flow? So we don't define/fetch them twice?

Reply:
Yeah, `approvalActions` are exported here, and used in `InvestActionsV2.vue`